nitrocli/src/main.rs

```diff
@@ -49,6 +51,19 @@ const RECV_TRY_COUNT: i8 = 40;
 const SEND_RECV_DELAY_MS: u64 = 200;
 
 
+/// Create an `error::Error` with an error message of the format `msg: err`.
+fn get_error(msg: &str, err: libnitrokey::CommandError) -> Error {
```

Comment:
Clippy has the following to say:
```
97 | fn get_error(msg: &str, err: libnitrokey::CommandError) -> Error {
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider taking a reference instead: `&libnitrokey::CommandError`
```
I think you may want to consider making `CommandError` derive from `Copy` (there is a `missing_copy...` lint), which should resolve this problem. Meanwhile, I think we should adhere to clippy's suggestion, unless you agree with the above and have enough changes piled up already to create a new release that would be consumes here (you don't have to do that right now, obviously).

Reply:
Unfortunately I currently can’t use clippy as it is not packaged for Debian and both the installation via `rustup` and manual compilation from source fail.  I’m happy to fix any issues you mention, but currently I can’t check them myself.

Regarding the specific error: I’d rather fix this here instead of waiting for a new `nitrokey` version. I’m also still thinking about ways to improve the error handling so this code may change anyway.